collector/src/compare.rs

```diff
@@ -7,14 +7,57 @@ use database::{
 };
 use tabled::{Table, Tabled};
 
+static ALL_METRICS: &[Metric] = &[
+    Metric::AssemblyFileSize,
```

Comment:
If we're not defaulting instructions, then I'd put them at the top of this list at least. The most useful metrics are instruction counts, cycles, walltime, max RSS and binary size.